packages/xchain-binance/src/client.ts

```diff
@@ -59,6 +58,18 @@ export type MultiSendParams = {
   memo?: string
 }
 
+/**
+ * Binance-specific Network enum that excludes 'stagenet' because
+ * @binance-chain has type checks limited to 'mainnet' and 'testnet',
+ * but `xchain-client` has 'mainnet', 'stagenet', and 'testnet'
+ *
+ * Remove this once @binance-chain has stagenet support.
+ */
+export enum Network {
+  Mainnet = 'mainnet',
+  Testnet = 'testnet',
+}
```

Comment:
No need to introduce another `Network` type, just map `stagenet` to `mainnet` internally (as you have already done in xchain-binance's client)